yubikey-u2f-pam-plugin/auth-pam-u2f.c

```diff
@@ -934,11 +934,12 @@ u2f_auth_verify(char *username, char* password, char* script_path, char **client
         buffer[br - 1] = '\0';
     }
     wait( &status);
+    close(pipefd[0]);
```

Comment:
This is the primary fix for the leaking file descriptor that causes issues over time.